diff --git a/v8/src/inspector/v8-runtime-agent-impl.cc b/v8/src/inspector/v8-runtime-agent-impl.cc
index b17432c..14ffeb6 100644
--- a/v8/src/inspector/v8-runtime-agent-impl.cc
+++ b/v8/src/inspector/v8-runtime-agent-impl.cc
@@ -1071,32 +1071,33 @@ void V8RuntimeAgentImpl::bindingCalled(const String16& name,
 }
 
 void V8RuntimeAgentImpl::addBindings(InspectedContext* context) {
-  int contextGroupId = context->contextGroupId();
-  int contextId = context->contextId();
-  const String16 contextName = context->humanReadableName();
-  if (!m_enabled) return;
-
-  protocol::DictionaryValue* globalBindings =
-      m_state->getObject(V8RuntimeAgentImplState::globalBindings);
-  if (globalBindings) {
-    for (size_t i = 0; i < globalBindings->size(); ++i) {
-      addBinding(context, globalBindings->at(i).first);
-      if (!m_inspector->getContext(contextGroupId, contextId)) return;
-    }
-  }
-
-  protocol::DictionaryValue* contextBindings =
-      m_state->getObject(V8RuntimeAgentImplState::contextBindings);
-  if (contextBindings) {
-    protocol::DictionaryValue* bindings =
-        contextBindings->getObject(contextName);
-    if (bindings) {
-      for (size_t i = 0; i < bindings->size(); ++i) {
-        addBinding(context, bindings->at(i).first);
-        if (!m_inspector->getContext(contextGroupId, contextId)) return;
-      }
-    }
-  }
+  return;
+  // int contextGroupId = context->contextGroupId();
+  // int contextId = context->contextId();
+  // const String16 contextName = context->humanReadableName();
+  // if (!m_enabled) return;
+  //
+  // protocol::DictionaryValue* globalBindings =
+  //     m_state->getObject(V8RuntimeAgentImplState::globalBindings);
+  // if (globalBindings) {
+  //   for (size_t i = 0; i < globalBindings->size(); ++i) {
+  //     addBinding(context, globalBindings->at(i).first);
+  //     if (!m_inspector->getContext(contextGroupId, contextId)) return;
+  //   }
+  // }
+  //
+  // protocol::DictionaryValue* contextBindings =
+  //     m_state->getObject(V8RuntimeAgentImplState::contextBindings);
+  // if (contextBindings) {
+  //   protocol::DictionaryValue* bindings =
+  //       contextBindings->getObject(contextName);
+  //   if (bindings) {
+  //     for (size_t i = 0; i < bindings->size(); ++i) {
+  //       addBinding(context, bindings->at(i).first);
+  //       if (!m_inspector->getContext(contextGroupId, contextId)) return;
+  //     }
+  //   }
+  // }
 }
 
 void V8RuntimeAgentImpl::restore() {
@@ -1229,7 +1230,7 @@ void V8RuntimeAgentImpl::inspect(
 }
 
 void V8RuntimeAgentImpl::messageAdded(V8ConsoleMessage* message) {
-  if (m_enabled) reportMessage(message, true);
+  // if (m_enabled) reportMessage(message, true);
 }
 
 bool V8RuntimeAgentImpl::reportMessage(V8ConsoleMessage* message,
diff --git a/v8/src/inspector/v8-runtime-agent-impl.h b/v8/src/inspector/v8-runtime-agent-impl.h
index b0efbe6..88bd126 100644
--- a/v8/src/inspector/v8-runtime-agent-impl.h
+++ b/v8/src/inspector/v8-runtime-agent-impl.h
@@ -160,7 +160,7 @@ class V8RuntimeAgentImpl : public protocol::Runtime::Backend {
                std::unique_ptr<protocol::DictionaryValue> hints,
                int executionContextId);
   void messageAdded(V8ConsoleMessage*);
-  bool enabled() const { return m_enabled; }
+  bool enabled() const { return false; }
 
  private:
   bool reportMessage(V8ConsoleMessage*, bool generatePreview);
