diff --git a/third_party/blink/renderer/core/dom/element.cc b/third_party/blink/renderer/core/dom/element.cc
index ecc99bd..368f77b 100644
--- a/third_party/blink/renderer/core/dom/element.cc
+++ b/third_party/blink/renderer/core/dom/element.cc
@@ -7852,6 +7852,11 @@ ShadowRoot* Element::OpenShadowRoot() const {
   return root && root->GetMode() == ShadowRootMode::kOpen ? root : nullptr;
 }
 
+ShadowRoot* Element::FakeShadowRoot() const {
+  ShadowRoot* root = GetShadowRoot();
+  return root;
+}
+
 ShadowRoot* Element::ClosedShadowRoot() const {
   ShadowRoot* root = GetShadowRoot();
   if (!root) {
diff --git a/third_party/blink/renderer/core/dom/element.h b/third_party/blink/renderer/core/dom/element.h
index 2ec1f99..7c1dcdf 100644
--- a/third_party/blink/renderer/core/dom/element.h
+++ b/third_party/blink/renderer/core/dom/element.h
@@ -1097,6 +1097,7 @@ class CORE_EXPORT Element : public ContainerNode {
     return HasShadowRoot() ? GetShadowRootInternal() : nullptr;
   }
   ShadowRoot* OpenShadowRoot() const;
+  ShadowRoot* FakeShadowRoot() const;
   ShadowRoot* ClosedShadowRoot() const;
   ShadowRoot* AuthorShadowRoot() const;
   ShadowRoot* UserAgentShadowRoot() const;
diff --git a/third_party/blink/renderer/core/dom/element.idl b/third_party/blink/renderer/core/dom/element.idl
index f145407..d689cc1 100644
--- a/third_party/blink/renderer/core/dom/element.idl
+++ b/third_party/blink/renderer/core/dom/element.idl
@@ -81,6 +81,7 @@ dictionary ParseHTMLUnsafeOptions {
 
     [RaisesException, MeasureAs=ElementAttachShadow] ShadowRoot attachShadow(ShadowRootInit shadowRootInitDict);
     [PerWorldBindings, ImplementedAs=OpenShadowRoot] readonly attribute ShadowRoot? shadowRoot;
+    [PerWorldBindings, ImplementedAs=FakeShadowRoot] readonly attribute ShadowRoot? fakeShadowRoot;
 
     [RaisesException] Element? closest(DOMString selectors);
     [RaisesException] boolean matches(DOMString selectors);
